### DISCUSSION TITLE
2FA Authentication for cloud

### DISCUSSION THREAD

**Discussion by mase1981** [koush/scrypted#1233] [Ideas]
Searched and was surprised to see no topic by that name. Would love to see support for 2FA authentication for cloud login.

**Comment by nkm8** [comment]
Agreed - not to bump, but it is very surprising to see 2023 on this request. I appreciate that using the suggest cloud configuration results in a double login requirement, but I would expect for something as sensitive as camera access to require at least one 2FA prompt.
